scripts/check-and-fix.sh

```diff
@@ -46,17 +46,17 @@ fi
 # Commit examples if outdated
 
 # Note: we need to add all files first so that new files are included in `git diff --cached` too.
-git add ./examples/compiled/vega_version ./examples/compiled/*.vg.json ./examples/compiled/*.svg ./examples/specs/normalized/*.vl.json
+git add ./examples/compiled/vega_version './examples/compiled/*.vg.json' './examples/compiled/*.svg' './examples/specs/normalized/*.vl.json'
```

Comment:
Why are these script fix a part of this follow fix?  

If they're not the same issue, I don't think we should merge them in the same PR?

Reply:
NVM -- it's good to combine them given they're a good test bed for this issue.